@kiva/kv-components/package.json

```diff
@@ -66,6 +66,7 @@
     "change-case": "^4.1.2",
     "date-fns": "^2.30.0",
     "embla-carousel": "^4.5.3",
+    "flag-icons": "^7.2.3",
```

Comment:
Why are we using this package instead of our fork of it: https://github.com/kiva/flag-icon-css ?

Reply:
We've already moved away from the fork in CPS as well for vue3/vite compatibility. This replicates the component in CPS.

I think I made a ticket in the PF queue to refactor and update our fork of the original forked flag repo if we deemed it necessary...